stumpy/aampdist_snippets.py

```diff
@@ -213,6 +237,8 @@ def aampdist_snippets(
     snippets_areas = np.empty(k)
     Q = np.full(D.shape[-1], np.inf)
     indices = np.arange(0, n_padded - m, m)
+    snippets_regimes = []
+    snippets_regimes = np.empty((0, 3), int)
```

Comment:
Why are we creating an empty list and then overwriting it with an empty array?

We should just stick to using an empty list and append arrays to the list and then once it is filled then we can convert it to an array all at once instead of appending to the array (as that is much slower)

Reply:
> Why are we creating an empty list and then overwriting it with an empty array?
My mistake

> We should just stick to using an empty list and append arrays to the list and then once it is filled then we can convert it to an array all at once instead of appending to the array (as that is much slower)

Didn't know that.